prov/efa/src/efa_base_ep.h

```diff
@@ -12,6 +12,10 @@
 #include "ofi_util.h"
 #include "rdm/efa_rdm_protocol.h"
 
+#if HAVE_EFADV_SL
+#define EFA_QP_DEFAULT_SERVICE_LEVEL 8
+#endif
```

Comment:
Nit: no need to make this conditional
```suggestion
#define EFA_QP_DEFAULT_SERVICE_LEVEL 8
```